db_helper.php

```diff
@@ -1,98 +1,129 @@
 <?php
+if (!file_exists('config.php')) {
```

Comment:
I'm not a fan of this. It is also not checked in the other files. If the config file is missing, it will crash and you see what happened. Would prefer if you remove this.